waku/v2/protocol/waku_filter/waku_filter.nim

```diff
@@ -48,30 +48,22 @@ proc notify*(filters: Filters, msg: WakuMessage, requestId: string = "") =
     # TODO: In case of no topics we should either trigger here for all messages,
     # or we should not allow such filter to exist in the first place.
     for contentFilter in filter.contentFilters:
-      if contentFilter.contentTopics.len > 0:
-        if msg.contentTopic in contentFilter.contentTopics:
+        if msg.contentTopic == contentFilter.contentTopic:
```

Comment:
Nitpicky, but we generally use 2-space indentation. :) I think this is 4 spaces.